Make.inc

```diff
@@ -381,6 +381,26 @@ else
 build_shlibdir = $(build_libdir)
 endif
 
+# If we are running on ARM, set certain options automatically
+ifneq (,$(findstring arm,$(ARCH)))
+LLVM_ASSERTIONS=1
+LLVM_FLAGS+="--with-cpu=cortex-a9 --with-float=hard --with-abi=aapcs-vfp --enable-targets=arm --enable-optimized --enable-assertions"
```

Comment:
--enable-optimized and --enable-assertions and --enable-targets shouldn't be in this list as they may break other parts of deps/Makefile

LLVM_ASSERTIONS should also generally not be set for the user